src/chrome/content/rules/NVidia.ru.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	For other NVidia coverage, see NVidia.xml.
+
+	Dropped:
+		- ^
+
+	504:
+		- www (except of /shield/)
+-->
+
+<ruleset name="NVidia.ru">
+	<target host="nvidia.ru" />
+	<target host="www.nvidia.ru" />
+	<target host="shield.nvidia.ru" />
+	<target host="store.nvidia.ru" />
+	<target host="support-shield.nvidia.ru" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://(www\.)?nvidia\.ru/shield/" to="https://www.nvidia.ru/shield/" />
+
+	<test url="http://nvidia.ru/shield/" />
+	<test url="http://www.nvidia.ru/shield/" />
```

Comment:
Please add an exclusion for `^http://nvidia\.ru/$` and `^http://www\.nvidia\.ru/$` and remove it from the whitelist.